clusterDeployer.py

```diff
@@ -259,6 +268,16 @@ def client(self) -> K8sClient:
             self._client = K8sClient(self._cc.kubeconfig)
         return self._client
 
+    def prepare_local_vms(self) -> None:
+        image_paths = {os.path.dirname(n.image_path) for n in self._cc.local_vms()}
+        for image_path in image_paths:
+            vp = VirshPool(
+                name=os.path.basename(image_path),
+                host=self._local_host.hostconn,
+                image_path=image_path,
+            )
+            vp.ensure_initialized()
+
```

Comment:
This only ensures pools are initialized on local host.  I think the right place to do this is in `ClusterHost._ensure_images()`.  That gets called for each host and there we already compute the exact image paths for each host.

Reply:
should be fixed.